src/cli/commands/config/replace.js

```diff
@@ -3,7 +3,7 @@ const debug = require('debug')
 const path = require('path')
 const log = debug('cli:config')
 log.error = debug('cli:config:error')
-const utils = require('utils')
+const utils = require('../../utils')
```

Comment:
No idea, I'll try to review the config commands later today, I think they're not working with the daemon running either :worried: